runtime_builders/publish_builders.py

```diff
@@ -32,6 +34,8 @@ def main():
                         required=True)
     args = parser.parse_args()
 
+    manifest = {}
```

Comment:
Yeah. At first I thought that maybe that would be fine (since when we update this we "republish" all the existing builder), but on second thought I'm not a big fan of that. Updated